src/main/features/event-management/views/eventManagement.njk

```diff
@@ -284,7 +284,7 @@
 <div class="govuk-grid-row ccs-page-section">
       <div class="govuk-grid-column-full">
       {% if eventType == 'FC' %}
-      {% if status == 'To Be Evaluated' %}
+      {% if status == 'to be evaluated' %}
```

Comment:
Please convert status to lower case because sometimes it will come in upper case then our condition will false.